src/envo/envo-edit.owl

```diff
@@ -31742,14 +31821,23 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001418> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_01001419> "Add more PATO qualities once they have been added to PATO."@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_01001419> <http://orcid.org/0000-0002-4366-3088>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasAlternativeId> <http://purl.obolibrary.org/obo/ENVO_01001419> "Cb")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_01001419> "Cumulonibus-Wolke"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_01001419> "Gewitterwolke"@de)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01001419> "envoAtmo")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01001419> "envoCloudAtlas"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01001419> "Cumulonimbus clouds have a very dark base which may be merged with low ragged clouds which frequently form underneath this cloud. The upper portion  may be smooth, fibrous, or striated. Cumulonimbus also contains large raindrops and, often, snowflakes, snow pellets or hailstones. The water droplets and raindrops may be substantially supercooled. Cumulonimbus is frequently associated with precipitation, sometimes in the form of virga. Cumulonimbus is often accompanied by lightning, thunder or hail. Cumulonimbus usually have their bases in the low level, but their vertical extent is often so great that their tops may reach into the middle and high levels."@en)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01001419> "cumulonimbus cloud"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001419> <http://purl.obolibrary.org/obo/ENVO_01001428>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001419> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> ObjectUnionOf(<http://purl.obolibrary.org/obo/ENVO_03520000> <http://purl.obolibrary.org/obo/ENVO_03520001> <http://purl.obolibrary.org/obo/ENVO_03520002>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001419> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_03520005> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0000407>))))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001419> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0000586>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01001419> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000086> <http://purl.obolibrary.org/obo/PATO_0001162>))
```

Comment:
Removing this as concentration isn't really the right concept - @raissameyer would you mind checking if "increased density" (and other density terms) are importable?

Reply:
PATO has [decreased](http://purl.obolibrary.org/obo/PATO_0001790) and [increased mass density](http://purl.obolibrary.org/obo/PATO_0001788) under [density](http://purl.obolibrary.org/obo/PATO_0001164). These terms are already part of the import script and available in ENVO. 

I'll add an axiom to connect cumulonimbus to increased mass density.

Reply:
Also, I noticed that the `'composed primarily of' some 'object aggregate' ...` axiom was off, and fixed that.